cppclean

```diff
@@ -88,6 +88,19 @@ def main():
                         help='add a header include path; '
                              'specify this multiple times for multiple '
                              'include paths')
+    parser.add_argument('--include-path-system', '-Is', action='append',
```

Comment:
Could you also remove the `-Is` and `-In` for the same reasoning from my previous post. A single letter option (or nothing at all) would be fine.